src/vs/base/common/comparers.ts

```diff
@@ -162,6 +233,54 @@ function compareAndDisambiguateByLength(collator: Intl.Collator, one: string, ot
 	return 0;
 }
 
+/** @returns `true` if the string is starts with a lowercase letter. Otherwise, `false`. */
+function isLower(string: string) {
+	const character = string.charAt(0);
+
+	return (character.toLocaleUpperCase() !== character) ? true : false;
+}
+
+/** @returns `true` if the string starts with an uppercase letter. Otherwise, `false`. */
+function isUpper(string: string) {
```

Comment:
I suggest to remove this util and instead use 
`isLowerAsciiLetter(s.charCodeAt(0))` and `isUpperAsciiLetter(s.charCodeAt(0))` from `strings.ts`

Reply:
Thanks for pointing out these functions. I hadn't found them previously though I did do a quick search. On review though, I think this might not be the best idea. It looks like `isLowerAsciiLetter()` and `isUpperAsciiLetter()` would only work for latin characters A through Z and would not work as intended for non-latin letters and letters with accents or diacritics. The utility functions I provided should work for all unicode lowercase and uppercase characters.

Reply:
@leilapearson makes sense, in that case I suggest to rename to `startsWithUpper` and `startsWithLower`